api/gin/webServer.go

```diff
@@ -159,14 +168,14 @@ func (ws *webServer) createMiddlewareLimiters() ([]shared.MiddlewareProcessor, e
 	return []shared.MiddlewareProcessor{sourceLimiter, globalLimiter}, nil
 }
 
-func (ws *webServer) sourceLimiterReset(reset resetHandler) {
-	betweenResetDuration := time.Second * time.Duration(ws.antiFloodConfig.SameSourceResetIntervalInSec)
+func sourceLimiterReset(ctx context.Context, reset resetHandler, sameSourceResetIntervalInSec uint32) {
```

Comment:
could have stayed as a method of webServer, then no need to pass it sameSourceResetIntervalInSec

Reply:
done